content/v2/csidriver/features/powerstore.md

```diff
@@ -484,7 +484,7 @@ Create a file called `config.yaml` and populate it with the following content
 ```
 
 **Note** :
-    skipCertificateValidation flag is currently unsupported. Please refrain from changing its value.
+    skipCertificateValidation flag is supported from driver version 1.15.
```

Comment:
Do we want to update this in older version of CSM ...if so, please mention the correct version of driver 1.15 is CSM version not driver version.